containers/home.tsx

```diff
@@ -1,66 +1,92 @@
-import React from 'react';
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import React, { useState, useEffect } from 'react';
+import Head from 'next/head';
+
+// Components
+import ListCardVideo from '../components/listCardVideo/listCardVideo';
+import ArrowPrevSVG from '../components/svg/arrowPrevSVG';
+import ArrowNextSVG from '../components/svg/arrowNextSVG';
+
+// Style
+import styles from '../styles/Home.module.css';
+interface iVideo {
+  cards?: [{ title: string; alt: any; src: any }];
+}
+
+const Home: React.FC<iVideo> = () => {
+  const [viewNextVideo, setViewNextVideo] = useState(0);
```

Comment:
Is the intention behind this to select the next video by its index? How about using its ID?

Reply:
Thanks, It's like you say, you mean rename the state for example as indexVideo / setIndexVideo?

Reply:
Yes, even something like `currentVideoIndex / setCurrentVideoIndex`.

Reply:
Sure, Thank you!